tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "lib": ["es6"],
+    "module": "commonjs",
+    "target": "es6",
+    "moduleResolution": "node",
+    "outDir": ".dist",
+    "strict": true,
+    "allowJs": true,
```

Comment:
Doesn't seem necessary. If you don't actually use `validate-token.js` make a branch at this commit to remember it is here and then remove it